src/tray.cpp

```diff
@@ -200,7 +200,7 @@ void TrayPrivate::notify(QModelIndex const & index, bool removing)
     {
         mConnectionsToNotify.removeOne(index);
         summary = Tray::tr("Connection lost");
-        body = Tray::tr("We have just lost the connection to %1 '%2'.");
+        body = Tray::tr("Lost the connection to %1 '%2'.");
```

Comment:
I'm not a native English speaker... but the body should be a "normal" sentence not just a "subject/heading"...

Reply:
Looks fine to me, it is no less a complete sentence without the redundant "we have just".

Reply:
OK... but, please, keep the style consistent, "Lost **the** connection..." vs. "Established connection..."